app/models/react_post.rb

```diff
@@ -1,6 +1,16 @@
 # ReactPost
 class ReactPost < ApplicationRecord
   # frozen_string_literal: true
+  attr_accessor :destroy_notify
```

Comment:
toàn bộ phần def + after create commit phải để sau belongs to
thư tự là 
- quan hê: belongs_to, has_many...
- validates
- delegate
- scope
- call back
- method